wire-tests-swift/no-manifest/src/main/swift/AllTypes.swift

```diff
@@ -1251,6 +867,21 @@ extension AllTypes : Sendable {
 extension AllTypes : ProtoExtensible {
 }
 
+/**
+ * Extensions of AllTypes
+ */
+extension AllTypes {
+
+    public var ext_opt_string: String? {
```

Comment:
This PR currently emits only STRING types for quicker testing. The rest of the fields should be added here once this plan is approved.